examples/piggy-bank/src/lib.rs

```diff
@@ -0,0 +1,19 @@
+#![no_std]
+
+use gstd::{debug, exec, msg};
+
+#[no_mangle]
+unsafe extern "C" fn handle() {
```

Comment:
`let value_available = exec::value_available()` and reuse it to avoid excess sys calls usage - it's cheaper

Reply:
Reworked.